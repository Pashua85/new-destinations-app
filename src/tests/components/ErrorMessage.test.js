import React from 'react';
import { shallow } from 'enzyme';
import ErrorMessage from '../../components/ErrorMessage';

test('should render ErrorMessage correctly', () => {
  const wrapper = shallow(<ErrorMessage errorMessage={'Test error message'} />);
  expect(wrapper).toMatchSnapshot();
})