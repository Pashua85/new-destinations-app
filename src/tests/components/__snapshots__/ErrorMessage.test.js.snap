// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ErrorMessage correctly 1`] = `
<div
  className="error-message"
>
  Test error message
</div>
`;
